tests/suites/0_stateless/02_0043_function_char.sql

```diff
@@ -0,0 +1,4 @@
+SELECT CHAR(97);
+SELECT CHAR(24929); -- 0x6161
```

Comment:
Add a case to cover the null case .

`select char(99,100,,null)`

Reply:
Do we treat NULL as valid input for char() function?

Reply:
Yes, if one of the arguments is null, the result is null.
But in function's eval, you don't have to care about null, it's already done by merging the argument's null bitmap after function's eval. So you can write anything you want(better to use default value) if it's null.